ports/zephyr/modules/_boot.py

```diff
@@ -54,7 +57,34 @@ def mount_all_disks():
     return retval
 
 
-if FlashArea and create_flash_partition():
+def mount_FS_flash():
```

Comment:
I suggest putting this function before `create_flash_partition()`, because that keeps the order they are called in.